prowler/providers/azure/services/iam/iam_custom_role_permits_administering_resource_locks/iam_custom_role_permits_administering_resource_locks.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "azure",
+  "CheckID": "iam_custom_role_permits_administering_resource_locks",
+  "CheckTitle": "Ensure a Custom Role is Assigned Permissions for Administering Resource Locks",
+  "CheckType": [],
+  "ServiceName": "iam",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "high",
+  "ResourceType": "AzureRole",
+  "Description": "Ensure a Custom Role is Assigned Permissions for Administering Resource Locks",
+  "Risk": "Resouce locks administration is a critical task that should be preformed from a custom role with the appropriate permissions. This ensures that only authorized users can administer resource locks.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": ""
+    },
+    "Recommendation": {
+      "Text": "Resource locks should be handled from a custom role",
+      "Url": ""
```

Comment:
The same again, we need to fill always the `Recommendation.Url`.

Reply:
Done!